api/v1alpha2/kubeadmbootstrapconfig_types.go

```diff
@@ -21,6 +21,12 @@ import (
 	kubeadmv1beta1 "sigs.k8s.io/cluster-api-bootstrap-provider-kubeadm/kubeadm/v1beta1"
 )
 
+type Format string
```

Comment:
We should add something like
`// +kubebuilder:validation:Enum=cloud-config`

Reply:
Done :)

Reply:
nice